### DISCUSSION TITLE
umi不配置fastRefresh也能热更新，请问配置上有什么额外作用吗？

### DISCUSSION THREAD

**Discussion by ILuZhangKun** [umijs/umi#7465] [Q&A]
最近在尝试mfsu，发现不是热更新失效，就是热更新有但是页面会pending到崩溃。查了很多issue都没有找到答案 ，最后发现把配置中的`fastRefresh: {}`注释掉就没问题了。
又尝试把`mfsu: {}` 和 `fastRefresh: {}`都注释掉，想看一下是否还有热更新，结果热更新也是有的呀。
所以比较好奇，配置fastRefresh的意义？

**Comment by xiaohuoni** [comment]
https://cloud.tencent.com/developer/article/1645453
可以看看这个
